cmd/attach/policy/cmd.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
```

Comment:
NP update the year :)
![what year is it](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNWlpMG5rOW9kZHl0MDRwdXFvZzRnMXVodWxxanBpaHZ0bHIzOW4zYiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/GcDtLf4RAdiRG/giphy.gif)

Reply:
fixed